dev-docs/FAQ.adoc

```diff
@@ -51,3 +51,37 @@ Run `gradle dev`, and then `cd ./packaging/build/dev/`.  Fire up your cluster
 via `bin/solr start -e cloud -noprompt` and then as you make changes to assets in `/solr/webapp/web`,
 run `gradle dev` to redeploy the web assets. Do a  hard refresh in your browser
 to pick up your changes.
+
+=== How do we ensure coding standards and quality?
+
+We use a number of tools for ensuring that Solr's codebase follows our community standards.  The most
```

Comment:
nit:  when writing adoc files, consider one sentence per line.  It helps with diffs.

Reply:
So, I guess we don't have a "we want 80 charaters more or less per line"?   We don't have lint on that today... I'll make the change.

Reply:
This isn't mandatory but is good-practice.  Cassandra did lots of such changes over the ref guide, and I'm grateful for it.